static/js/TopicLandingPage/TopicLandingSearch.jsx

```diff
@@ -22,22 +29,31 @@ const getSuggestions = async (input) => {
     }
 
     const _parseSuggestions = (completionObjs, lang) => {
-      let topics = [];
-      if (completionObjs.length > 0) {
-        topics = completionObjs.map((e) => ({
-          title: `#${_capitalizeFirstLetter(e.title)}`,
-          categoryPathTitle: `${_getFormattedPath(e.key, lang)}`,
+      if (completionObjs.length === 0) return [];
```

Comment:
Our style is to always use `{}` in if-statements or for-loops, even if they're one line. This avoids bugs if the if-statement grows to be more than one line.